sentry/src/main/java/io/sentry/instrumentation/file/SentryFileInputStream.java

```diff
@@ -112,6 +112,7 @@ public long skip(final long n) throws IOException {
   @Override
   public void close() throws IOException {
     spanManager.finish(delegate);
+    super.close();
```

Comment:
I don't think we need to do this, as SentryFileInputStream isn't backed by any real inputstream, but rather the delegate is used - but it shouldn't hurt, right @romtsn ?

Reply:
Yes, I wanted to leave the same comment - I'd rather remove these `super` calls, not sure if they will mess up something, but since we're using a delegate, it should be fine. E.g. apache-commons also don't call super, but just delegate https://github.com/apache/commons-io/blob/43d3ac6cbdb419a1929870b04be181da20be7f03/src/main/java/org/apache/commons/io/input/ProxyInputStream.java#L119-L122

Reply:
agree, was just thinking if strict mode could have a check on those close() functions, but it seems it doesn't.
Looks safe to remove it